### PR TITLE
docs: warn developer that bearer is written with ES6

### PR DESCRIPTION
**Pull request by tanguyantoine**
## 🐻 What your PR is doing?

update documentation so that people know they need to update their bundle config to have bearer writtend with ES5 syntax

## 📦 Package concerned

- @bearer/js
- @bearer/react

### CODE REVIEW

File: packages/js/README.md [hunk 1]
```diff
@@ -41,6 +41,8 @@ class MyApp {
 }
 ```
 
+**:warning: `@bearer/js` is written in ES6**. If you want to support older browser (IE11) you'll have to add extra configuration to your bundling system. You'll need to explicitly tell your bundler (ex: webpack) to process and transform bearer's files.
+
```
Comment (cfabianski) [inline packages/js/README.md]: Can we provide an example?

Comment (tanguyantoine) [reply packages/js/README.md]: we can. we would have to do it for webpack and other bundlers. maybe a link to the wiki where we can update it without having to update the codebase

Comment (tanguyantoine) [reply packages/js/README.md]: here it is: https://github.com/Bearer/bearer/wiki

**Comment by tanguyantoine** [comment]
superseeded by #727

**Pull request closed without merging**